CodeStarterCamp_Week4/Exercise/exercise.swift

```diff
@@ -7,29 +7,38 @@
 
 import Foundation
 
+
+
 struct Exercise {
     let name: String
-    let action:() -> Void
+    let action: () -> Void
+    
+    enum ExerciseLevel {
+        static let highIntensity = 20 ... 30
+        static let mediumIntensity = 10 ... 20
+        static let lowIntensity = 5 ... 10
+    }
```

Comment:
사용할 범위들을 열거형으로 구현해 하드코딩을 최소화하셨네요 👍
밀접한 관련이 있다 생각해 열거형을 Exercise 타입 내부에 위치시키셨군요! 좋네요
네이밍은 명확하고 좋지만 사용할 때 `Exercise.ExerciseLevel.highIntensity`등으로 접근하기보다
`Exercise.Level.high` 혹은 `Exercise.Intensity.high` 등으로 접근할 수 있도록 구현해도 충분히 명확할 것 같아요!
```suggestion
    enum Level {
        static let high = 20 ... 30
        static let medium = 10 ... 20
        static let low = 5 ... 10
    }
```